*~ are backup files too, created by vim in certain configurations

--- util-vserver-0.30.214/scripts/functions~	2008-04-29 17:21:44.000000000 +0300
+++ util-vserver-0.30.214/scripts/functions	2008-04-29 17:21:45.490461173 +0300
@@ -73,7 +73,7 @@
     test ${2:--f} "$1" || return 1
 
     case $1 in
-	(*.rpmsave|*.rpmnew|*.rpmorig|*.cfsaved*|*.~*~)	return 1;;
+	(*.rpmsave|*.rpmnew|*.rpmorig|*.cfsaved*|*~)	return 1;;
     esac
 
     return 0
